dashboards/kiwi/sqs-metrics.json

```diff
@@ -3,371 +3,406 @@
     "configurationVersions": [
       7
     ],
-    "clusterVersion": "1.293.129.20240615-053240"
+    "clusterVersion": "1.303.42.20241021-231645"
   },
   "dashboardMetadata": {
     "name": "KIWI - SQS - Metrics",
     "shared": false,
```

Comment:
This will need changing to **true**, otherwise the dashboard won't be visible once deployed into Dynatrace